sdk/identity/identity/README.md

```diff
@@ -30,6 +30,14 @@ npm install --save @azure/identity
 - An Azure subscription.
   - You can sign up for a [free account](https://azure.microsoft.com/free/).
 - The [Azure CLI][azure_cli] can also be useful for authenticating in a development environment, creating accounts, and managing account roles.
+- [node-gyp](https://github.com/nodejs/node-gyp/blob/master/README.md#installation), which means:
```

Comment:
Isnt this only if one wants to use the persistent cache feature? 
And in that case shouldnt we also call out the msal-extensions needing to be installed?

Reply:
I think this got messed up somehow. I'll clean it up.

Reply:
I think that I had some local changes that I didn't commit, so when I pulled the other PR my local changes messed up with what was in the PR. Anyway, I manually copied the files as they were on the previous PR into this PR. This PR should be clean now.